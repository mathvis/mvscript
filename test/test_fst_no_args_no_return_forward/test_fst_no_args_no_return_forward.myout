Decl (FunctionDef (FunctionIdentifier "a") [] void Nothing)

--- STATE ---

Function symbol table:
	Function name: a
		Return type: void
		Arguments: 
		Has body: False
