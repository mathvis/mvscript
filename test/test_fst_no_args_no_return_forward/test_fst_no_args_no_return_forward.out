(Decl (FunctionDef (FunctionIdentifier "a") [] VoidT Nothing),ParserState {config = Configuration {debug = True, collapseOperations = True, collapseControlFlow = True, colors = True}, st = fromList [], fst = fromList [("a",FunctionData {returnType = VoidT, arguments = [], hasBody = False})], context = Nothing, unresolvedFunctionCalls = []})
